composer.json

```diff
@@ -23,12 +23,15 @@
         "twig/twig": "~1.12",
         "knplabs/knp-markdown-bundle": "~1.4",
         "egeloen/ckeditor-bundle": "~2.2",
+        "sonata-project/admin-bundle": "~2.3",
         "sonata-project/block-bundle": "~2.2,>=2.2.1",
-        "sonata-project/core-bundle": "~2.3,>=2.3.1"
+        "sonata-project/core-bundle": "~2.3,>=2.3.1",
```

Comment:
if you put this to the end, it would not occur as a change, because of the colon